docs/framework/data/adonet/ef/language-reference/ref-entity-sql.md

```diff
@@ -31,7 +31,7 @@ REF( expression )
   
 2. Pass the following query as an argument to the `ExecutePrimitiveTypeQuery` method:  
   
- [!code-csharp[DP EntityServices Concepts 2#REF](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts 2/cs/entitysql.cs#ref)]  
+ [!code-sql[DP EntityServices Concepts 2#REF](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts 2/cs/entitysql.cs#ref)]  
```

Comment:
Changing just the type didn't work. Let's just as follows:

```suggestion
 [!code-sql[DP EntityServices Concepts 2#REF](~/samples/snippets/tsql/VS_Snippets_Data/dp entityservices concepts 2/tsql/entitysql.sql#ref)]  
```

Reply:
See the comment below. I made this change and changes to all other files in this PR with a similar line, but it resulted in link warnings.

Reply:
See comments below. I made the suggested changes.